localstack/services/s3/provider.py

```diff
@@ -1874,7 +1874,7 @@ def s3_response_is_delete_keys(fn, self):
         """
         return get_safe(self.querystring, "$.x-id.0") == "DeleteObjects" or fn(self)
 
-    @patch(moto_s3_responses.S3ResponseInstance.parse_bucket_name_from_url, pass_target=False)
+    @patch(moto_s3_responses.S3Response.parse_bucket_name_from_url, pass_target=False)
```

Comment:
this should have been a sign from the big issue this bump is fixing, but well 🤷‍♂️ fixing the patch to properly work